third_party/3/jack/__init__.pyi

```diff
@@ -0,0 +1,265 @@
+from typing import Any, Callable, Dict, Generator, Iterable, Iterator, List, Optional, Sequence, Tuple, Union, overload
+
+import numpy as np
+
+JackPositionT = Any
+
+class CBufferType:
```

Comment:
This only exists because I couldn't for the life of me find the type definition for cffi's `buffer` type; if someone can think of a better way of doing this I'd appreciate it!

Reply:
See perhaps python/typing#593 (summary: there's no good way yet).